pkg/login/printClusterInfo.go

```diff
@@ -0,0 +1,71 @@
+package login
+
+import (
+	"fmt"
+
+	"github.com/openshift/backplane-cli/pkg/ocm"
+	logger "github.com/sirupsen/logrus"
+)
+
+//displayClusterInfo retrieves and displays basic information about the target cluster.
+
+func PrintClusterInfo(clusterID string) error {
+	logger := logger.WithField("clusterID", clusterID)
+
+	// Retrieve cluster information
+	clusterInfo, err := ocm.DefaultOCMInterface.GetClusterInfoByID(clusterID)
+	if err != nil {
+		return fmt.Errorf("error retrieving cluster info: %w", err)
+	}
+
+	// Display cluster information
+	fmt.Printf("\n%-25s %s\n", "Cluster ID:", clusterInfo.ID())
```

Comment:
May be better to refactor the repeated code of `fmt.Printf("%-25s ...")` into a helper function like:
```
func printClusterField(fieldName string, value interface{}) {
    fmt.Printf("%-25s %v\n", fieldName, value)
}
```
so you can use it like:
```
printClusterField("Cluster ID:", clusterInfo.ID())
printClusterField("Cluster Name:", clusterInfo.Name())
...
```